### PR TITLE
Release tweaks

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -49,6 +55,7 @@ The resulting localized resources are added/updated in the repository periodical
 ### Post Release
 
 - [ ] Publish blog post
+- [ ] Update news files
```
Comment (thc202) [inline RELEASING.md:58]: `, if needed.`? zapbot already updates them.

Comment (psiinon) [reply RELEASING.md:58]: True, although I like the "Welcome to X" news item for the latest release - will tweek..

Comment (psiinon) [reply RELEASING.md]: Tweaked..

**Review by kingthorin — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**